examples/migraphx/custom_op_hip_kernel/custom_op_hip_kernel.cpp

```diff
@@ -0,0 +1,91 @@
+#include <algorithm>
+#include <hip/hip_runtime.h>
+#include <migraphx/migraphx.hpp> // MIGraphX's C++ API
+#include <numeric>
+
+#define TENSOR_SIZE 3 * 3
```

Comment:
This should just be a const variable.

Reply:
removed that. calculating elements from shape.